modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -425,21 +427,30 @@ sda.enkf.multisite <- function(settings,
         writeLines(runs.tmp[runs.tmp != ''], file.path(rundir, 'runs.txt'))
         paste(file.path(rundir, 'runs.txt'))  ## testing
         Sys.sleep(0.01)                       ## testing
-        PEcAn.workflow::start_model_runs(settings, write=settings$database$bety$write)
+        PEcAn.workflow::qsub_parallel(settings, files=PEcAn.workflow::merge_job_files(settings, 10), prefix = paste0(obs.year, ".nc"))
         
         #------------- Reading - every iteration and for SDA
         
         #put building of X into a function that gets called
-        reads <- build_X(out.configs = out.configs, 
-                         settings = settings, 
-                         new.params = new.params, 
-                         nens = nens, 
-                         read_restart_times = read_restart_times, 
-                         outdir = outdir, 
-                         t = t, 
-                         var.names = var.names, 
-                         my.read_restart = my.read_restart,
-                         restart_flag = restart_flag)
+        max_t <- 0
+        while (is.character(try(reads <- PEcAnAssimSequential:::build_X(out.configs = out.configs, 
+                                                                        settings = settings, 
```

Comment:
excess whitespace makes the code hard to read. Generally you want to align arguments vertically and keep lines to <80 characters wide (including whitespace)

Reply:
Fixed.